### ISSUE TITLE
Victim of Staff Abuse

### ISSUE THREAD

**Issue by ugo6ftbelow** [Calytic/cyclone.rust#2]
So on 6/5/15 I was roaming the map and seen someone building a base, had no idea who it was, and was in need of supplies so I crept in to a bush, waited patiently for them to get closer, shot them in the head and ran up to find they had a whopping 99k metal frags on them.  I grabbed the frags quickly but was immediately shot by someone...luckily they was far away enough and I had enough food to recover from being down...i ran into a bush, hid...and they never found me...I teleported away, then my game got dc'd (first sign of staff abuse).  This should of been a good payday had I not later been the victim of more staff abuse.  I know I was because the person I robbed was SOON, and the Staff on that night was NYCBASEBALLER (his friend that gave him the metal frags to begin with for free!).  Right before logging off for the evening I seen nycbaseballer mention in chat something about someone being stupid with 100k metal frags they had robbed someone of...and I mentioned that I had just robbed some poor soul working in a field of 100k metal frags.  He then told me to bank it...I said "na" and then proceeded to split it up between a 2 boxes I had hid in fields in bushes.  Well I awake the next day and the bushes around are literally gone like staff was watching and was looking.  The box was out in the open now, bush gone, and was emptied, the staff not only took the metal frags but also took, stones, timber, and an incendiary rocket they never found to begin with.  This is true staff abuse at its finest.  I found out from another clan that the staff was helping Soon rebuild a glitched base and I came through and messed it up, but like I knew...wtf...like you guys dont have enough metal frags you have to mess with new players like this?  Also to make it this obvious to despawn bushes and leave the boxes out in the open like that.  Very sloppy..very abusive.   Not sure I want to continue with your server or even support it like I was going to with an donation from an upcoming paycheck.  Especially without an explanation as to why NYCbaseballer decided to spy on me and mess with me like this without just explaining to me he was helping Soon and I came through and messed with him.  Very poor ethics from staff I thought was there to help the gamers playing and paying to play on this server!  You've lost respect!  I'd like an apology from NYCbaseballer, the metal frags and incendiary rocket back asap!

**Comment by Calytic** [comment]
I apologize on behalf of the entire Cyclone team that you had a bad experience on our server.

The DC you are referring to was part of a bug due to half-blocks.  Everyone was crashing, including admins, for a period of 7 hours after last weeks patch.  I have spoken with NYC and the metal you are referring to was given to SOON as compensation for a bug - not favoritism.  NYC is not collaborating with players and staff involvement in the game is strictly prohibited.

That night after the patch was tough for everyone due to the CTD bugs.

We will not be compensating you for allegedly losing items to Staff involvement because a) there is no proof and b) Staff know better than to do what you are describing.

**Comment by Calytic** [comment]
I have confirmed with Soon that NYC never gave him anything at all.

**Comment by nycbaseballer** [comment]
Hello ugo6ftbelow,

I am sorry you thought I was abusing my powers and making you feel disrespected.  I for one take this job very seriously.  I do not hand out people free items without Calytics' permission.

In this case with Soon, I was told by Calytic to upgrade his base.  Soon's base was the victim of this bug where people's rust client crashed and returned them to the window screen.  Calytic had to delete his base and in return to compensate him for this trouble, was told to upgrade his base that he built from twig. As I was upgrading his base, my rust client crashed and I spoke to Calytic on Ventrillo about my crashing issue and told him I was done for the night.  Roughly 2 hours later, I signed back on to find myself killed by [LOL] Leeter.  I for one did not troll you, nor did I take your belongings. I know better than that.  I do not despawn or spawn anything on this server as to I don't have permission.

Again, I do apologize that you thought I did something wrong and I am glad you brought this up and hopefully this will make it clear to you.
